packages/terra-doc-template/examples/test-examples/DefaultDocTemplate.example.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+import DocTemplate from '../../src/DocTemplate';
+import readme from './TestReadme.md';
+// eslint-disable-next-line import/no-duplicates
+import TestComponentExample from './TestComponentExample';
+// eslint-disable-next-line import/no-webpack-loader-syntax, import/first, import/no-unresolved, import/extensions, import/no-duplicates
+import exampleSrc from '!raw-loader!./TestComponentExample.jsx';
+// eslint-disable-next-line import/no-webpack-loader-syntax, import/first, import/no-unresolved, import/extensions
+import testComponentSrc from '!raw-loader!./TestComponent.jsx';
+
+const Index = () => {
+  const propsTables = [{ componentSource: testComponentSrc, componentName: 'Test Component' },
+    { componentSource: testComponentSrc, componentName: 'Test Component (Again)' }];
```

Comment:
nit: This is up to your preference, I think multi-line array syntax is a little easier to read when formatted similar to this:

```
const propsTables = [
  { componentSource: testComponentSrc, componentName: 'Test Component' },
  { componentSource: testComponentSrc, componentName: 'Test Component (Again)' },
];
```